packages/chat/tests/messageUtils.test.ts

```diff
@@ -0,0 +1,110 @@
+			'is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a',
+			'test This is a test This is a test This is a test'
+		]);
+	});
+
+	test('Over character limit and should have 4 messages', () => {
+		expect(autoSplitterWithSpaces(testString.repeat(100), 500)).toStrictEqual([
+			'This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This',
+			'is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a',
+			'test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test'
+		]);
+	});
+
+	test('Over character limit, has no spaces and should have 2 messages', () => {
+		expect(autoSplitterWithSpaces(testStringNoSpaces.repeat(40), 250)).toStrictEqual([
+			'ThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisat',
+			'estThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatestThisisatest'
+		]);
+	});
+
+	test('Over character limit, has no spaces and should have 2 messages', () => {
```

Comment:
Again, probably doesn't test anything the previous test didn't already test.

Reply:
@d-fischer Wouldn't it test that it does indeed split even if there is no space? In practice it would be unlikely, so it's understandable if you think we shouldn't include it

Reply:
Nevermind, I see you were referring to the previous test in the code not in the comments